trace/Span.md

```diff
@@ -1,8 +1,11 @@
-# Span API
+# Span
 
-This class represents a Span. A span contains a SpanContext and  allows users to record 
-tracing events based on the data model defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto).
-All Span implementations MUST not allow users to read the recorded data, 
+Span represents a single operation within a trace. Spans can be nested to form a trace tree. 
+Often, a trace contains a root span that describes the end-to-end latency and, optionally, one or
```

Comment:
This is hard to follow. You may explain traces are a tree. There is a root span and many nodes that represents worked started from the root point.